librad/src/git/types/reference.rs

```diff
@@ -367,6 +370,39 @@ impl<N, R> Reference<N, R, Many> {
             namespace: namespace.into(),
         }
     }
+
+    /// Build a reference that points to:
+    ///     * `refs[/namespaces/<namespace>]/refs[/remotes/<remote>]/rad/*`
```

Comment:
The same suggestions go for the rest of them, I believe. We would have `refs/refs` if the namespace was optional :)